shared_model/validators/transaction_validator.hpp

```diff
@@ -201,7 +202,7 @@ namespace shared_model {
           const detail::PolymorphicWrapper<interface::SubtractAssetQuantity>
               &saq) const {
         ReasonsGroupType reason;
-        reason.first = "SubtractAssetQuantity";
+        addInvalidCommand(reason,"SubtractAssetQuantity");
```

Comment:
Space between `,` and `"SubtractAssetQuantity"` is missing. Maybe run clang_format?

Reply:
Just use `clang-format`